--- nm-applet-pre-dbus-126.conf.old 
+++ nm-applet-pre-dbus-126.conf 
@@ -11,7 +11,7 @@
 		<allow send_destination="org.freedesktop.NetworkManagerUserSettings"
 		       send_interface="org.freedesktop.NetworkManagerSettings.Connection.Secrets"/>
         </policy>
-        <policy at_console="true">
+        <policy group="plugdev">
                 <allow own="org.freedesktop.NetworkManagerUserSettings"/>
 
                 <allow send_destination="org.freedesktop.NetworkManagerUserSettings"/>
