diff -urNad tuxcmd-0.6.50~/UCore.pas tuxcmd-0.6.50/UCore.pas
--- tuxcmd-0.6.50~/UCore.pas	2008-08-24 23:33:52.000000000 +0200
+++ tuxcmd-0.6.50/UCore.pas	2009-02-07 10:39:58.000000000 +0100
@@ -2381,11 +2381,12 @@
   //  Get mount name
   mntent := getmntent(fd);
   while mntent <> nil do begin
-    DebugMsg(['FillDefaultFstabMounterItems: found "', mntent^.mnt_dir, '" --> "', mntent^.mnt_fsname, '", fstype ', mntent^.mnt_type]);
-    if (mntent^.mnt_dir <> '/') and (mntent^.mnt_dir <> '/boot') and (Pos('/proc', mntent^.mnt_dir) <> 1) and
+    DebugMsg(['FillDefaultFstabMounterItems: found "', mntent^.mnt_dir, '" --> "', mntent^.mnt_fsname, '", fstype "', mntent^.mnt_type, '"']);
+    if (mntent^.mnt_dir <> nil) and (mntent^.mnt_type <> nil) and (strlen(mntent^.mnt_dir) > 0) and (strlen(mntent^.mnt_type) > 0) and
+       (mntent^.mnt_dir <> '/') and (mntent^.mnt_dir <> '/boot') and (Pos('/proc', mntent^.mnt_dir) <> 1) and
        (Pos('/dev', mntent^.mnt_dir) <> 1) and (Pos('/sys', mntent^.mnt_dir) <> 1) and (mntent^.mnt_dir <> 'swap') and
        (mntent^.mnt_type <> 'swap') and (mntent^.mnt_type <> 'rpc_pipefs') and (mntent^.mnt_type <> 'none') and
-       (mntent^.mnt_dir <> 'none') and (mntent^.mnt_dir <> '') then
+       (mntent^.mnt_dir <> 'none') then
     begin
       MounterItem := TMounterItem.Create;
       MounterItem.DisplayText := '';
